.github/workflows/hello-world-ci.yml

```diff
@@ -0,0 +1,23 @@
+name: hello-world-ci
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./submissions/hello_world
+    strategy:
+      matrix:
+        python-version: [3.8]
+
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
Use the newer checkup action (the other workflow uses v3)

Reply:
Updated checkout to `v3` and setup-python to `v4`